app/src/test/java/org/standardnotes/notes/CryptTest.kt

```diff
@@ -5,21 +5,4 @@ import org.junit.Test
 import org.standardnotes.notes.comms.Crypt
 import org.standardnotes.notes.comms.data.AuthParamsResponse
 
-class CryptTest {
-
-    @Test
-    fun isParamsSupported_ifSha512_returnsTrue() {
-        val params: AuthParamsResponse = AuthParamsResponse()
-        params.pwAlg = "sha512"
-
-        assertEquals(true, Crypt.isParamsSupported(params))
-    }
-
-    @Test
-    fun isParamsSupported_ifNotSha512_returnsFalse() {
-        val params: AuthParamsResponse = AuthParamsResponse()
-        params.pwAlg = "GO UCLA BRUINS"
-
-        assertEquals(false, Crypt.isParamsSupported(params))
-    }
-}
\ No newline at end of file
+class CryptTest
```

Comment:
would be nice to add some tests later. `createContentEncrypted`, `generateEncryptedKey`, come to mind